### PR TITLE
Log data used to generate ProgramListView keys

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

File: course_discovery/apps/api/cache.py [hunk 1]
```diff
@@ -78,23 +79,27 @@ def process_cache_response(self, view_instance, view_method, request, args, kwar
             kwargs=kwargs
         )
         response_triple = self.cache.get(key)
+        logger.info(view_instance.__class__.__name__)
```
Comment (mikix) [inline course_discovery/apps/api/cache.py]: Is this supposed to be here?

**Review by mikix — APPROVED** [review]
You're really digging into this section of code, eh? :)

File: course_discovery/apps/api/cache.py (hunk 1 above)
Comment (cpennington) [reply course_discovery/apps/api/cache.py]: Nope.

**Comment by cpennington** [comment]
I mean, something is definitely bizarre around this code, to be triggering as many cache misses as we're seeing.

**Comment by mikix** [comment]
> I mean, something is definitely bizarre around this code, to be triggering as many cache misses as we're seeing.

Yeah for sure, wasn't suggesting focus is bad. Just this investigation has you very in the weeds. You'll be able to finish this ticket with an edX degree in `cache.py`

**Pull request merged**